internal/cloudfront/distribution.go

```diff
@@ -54,6 +55,17 @@ type loggingConfig struct {
 	Prefix        string
 }
 
+// CustomBehaviors returns a slice of all custom Behavior sorted by descending path length
+func (d Distribution) CustomBehaviors() []Behavior {
```

Comment:
A wise man (you :P) once told me the caller doesn't need to know how we're doing it, just what we're doing (or something along those lines, just paraphrasing). 

That said, since the "sorted" part is actually a business rule, not just an implementation detail, I think it's better if we can express that in code, not just documentation. I'll change it, thanks!

Reply:
show